app/src/components/domain/DrefExportModal/index.tsx

```diff
@@ -57,6 +57,7 @@ function DrefExportModal(props: Props) {
                 export_id: id,
                 export_type: type,
                 selector: '#pdf-preview-ready',
+                per_country: undefined,
```

Comment:
@frozenhelium any reason to do this?

Reply:
`per_country` is apparently a required field according to the types

Reply:
@k9845 Let's ensure `per_country` is optional for DREF export